src/pages/index.js

```diff
@@ -9,7 +9,7 @@ const Wrapper = styled.section`
   align-items: center;
   display: flex;
   flex-direction: column;
-  margin: 20px 0 0;
+  margin: 90px 0 0;
```

Comment:
This was done since the top nav is now fixed to the top of the viewport even as you scroll, so all page content has to have at least 70px from the top so as not to hide content behind the top nav